### PR TITLE
Added attribute span property for fileMemberElement

### PR DESCRIPTION
**Pull request by akshita31**
Issue : https://github.com/OmniSharp/omnisharp-vscode/issues/429

Added properties to contain the attribute span in a file member element, which will be used by the editor to display the "# of references" before the attributes.

### CODE REVIEW

**Comment by akshita31** [comment]
Omnisharp-vscode side : https://github.com/OmniSharp/omnisharp-vscode/pull/1938

File: src/OmniSharp.Abstractions/Models/v1/MembersTree/FileMemberElement.cs [hunk 1]
```diff
@@ -9,6 +9,10 @@ public class FileMemberElement : IComparable<FileMemberElement>
 
         public QuickFix Location { get; set; }
 
+        public int AttributeSpanStart { get; set; }
```
Comment (rchande) [inline src/OmniSharp.Abstractions/Models/v1/MembersTree/FileMemberElement.cs]: Can we make these some kind of range or span type?

Comment (akshita31) [reply src/OmniSharp.Abstractions/Models/v1/MembersTree/FileMemberElement.cs]: We could pass both AttributeSpanStart and AttributeSpanEnd in a Span type class. But in that case we will also have to define this class in protocol.ts in omnisharp-vscode. Should that be done ?

Comment (rchande) [reply src/OmniSharp.Abstractions/Models/v1/MembersTree/FileMemberElement.cs]: Protocol.ts defines a range type: https://github.com/OmniSharp/omnisharp-vscode/blob/master/src/omnisharp/protocol.ts#L450

Comment (akshita31) [reply src/OmniSharp.Abstractions/Models/v1/MembersTree/FileMemberElement.cs]: @rchande Yes it does, but the range has the start and end as `Point` type. And I couldn't spot a function in omnisharp-roslyn that converts from a `span` to `range`

**Pull request merged**